### PR TITLE
Modify AirQualityViewModel

### PR DESCRIPTION
**Pull request by irinkadat**
Optimise AirQuality ViewModel to better adjust MVVM architecture pattern

### CODE REVIEW

File: CollabProject/Scenes/AirQuality/AirQualityViewController.swift [hunk 1]
```diff
@@ -7,24 +7,23 @@
 
 import UIKit
 
-class AirQualityViewController: UIViewController {
-    private var viewModel = AirQualityViewModel()
+class AirQualityViewController: UIViewController, AirQualityViewModelDelegate {
```
Comment (begsense) [inline CollabProject/Scenes/AirQuality/AirQualityViewController.swift:11]: add //MARKS are required

**Review by begsense — COMMENTED** [review]
marks for irinka's branch

File: CollabProject/Scenes/AirQuality/AirQualityViewController.swift (hunk 1 above)
Comment (irinkadat) [reply CollabProject/Scenes/AirQuality/AirQualityViewController.swift:11]: I'll fix it

**Review by begsense — APPROVED** [review]
marks added

**Review by tamokakhidze — APPROVED** [review]
All good

**Pull request merged**